src/components/FullTextView/FullTextView.spec.tsx

```diff
@@ -32,6 +33,22 @@ describe('FullTextView', () => {
     expect(component.find('[data-hook="popover-content"]').at(0).text()).toBe('Delete this super awesome thing?');
   });
 
+  it('should dynamically load and display full content on hover and hide it on leave in tooltip using `shouldLoadAsync` prop', async () => {
+    const content = (
+      <div>
+        <span>Delete this super awesome thing</span>
+        <i>?</i>
+      </div>
+    );
+    const component = mount(<FullTextView shouldLoadAsync maxWidth={0}>{content}</FullTextView>);
```

Comment:
any reason to use enzyme here and not `createDriver`?

Reply:
It's the way we tested it before my PR, I just created 1 more test case based on already existed.